src/DestroyCommand.php

```diff
@@ -21,9 +21,8 @@ protected function configure()
     /**
      * Execute the command.
      *
-     * @param  \Symfony\Component\Console\Input\InputInterface  $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
-     * @return void
```

Comment:
Passing any parameters to php-cs-fixer makes it avoid using the config file. Don't pass `src`.